images/kubeflow-pipelines/configs/latest.cache-deployer.apko.yaml

```diff
@@ -0,0 +1,31 @@
+contents:
+  packages:
+    - kubeflow-pipelines-cache-deployer
+    - kubeflow-pipelines-cache-deployer-compat
+    - busybox
+    - bash
+    - kubectl-latest
+    - openssl
```

Comment:
these are bordering on enough _required_ dependencies to hoist into the melange config instead 🙏

Reply:
makes sense, will do, thanks!